config-connector/solutions/projects/kpt/project-services/Kptfile

```diff
@@ -0,0 +1,22 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: project-services
+packageMetadata:
+  shortDescription: enables a service on a project 
+openAPI:
+  definitions:
+    io.k8s.cli.setters.service-name:
+      description: API service name 
+      x-k8s-cli:
+        setter:
+          name: service-name
+          value: pubsub.googleapis.com
+          setBy: package-default
+    io.k8s.cli.setters.project-id:
+      description: ID of project to enable service for
+      x-k8s-cli:
+        setter:
+          name: project-id
+          value: ${PROJECT_ID}
```

Comment:
Should it be "${PROJECT_ID?}" (maps to the value in the yaml file)?

Reply:
Yes, it should. Good catch.